proto/webhook.proto

```diff
@@ -25,6 +25,11 @@ message _Webhook {
   _WebhookDestination destination = 3;
 }
 
+message _WebhookSecret {
```

Comment:
`_WebhookItem` or something? That way the name is not tied to what fields are declared in the struct. Not a strong opinion. Either way the names that will show in the SDKs will matter more.